File: Pod/Classes/PURLoggerConfiguration.m
#import "PURLoggerConfiguration.h"
#import "PURLogStore.h"

@implementation PURLoggerConfiguration

+ (instancetype)defaultConfiguration
{
    PURLoggerConfiguration *configuration = [PURLoggerConfiguration new];
    configuration.logStore = [PURLogStore defaultLogStore];
    configuration.filterSettings = @[];
    configuration.outputSettings = @[];

    return configuration;
}

@end

File: Pod/Classes/PURLoggerConfiguration.swift
//
//  PURLoggerConfiguration.swift
//  Pods
//
//  Created by admin on 7/27/17.
//
//

//  Converted with Swiftify v1.0.6414 - https://objectivec2swift.com/
import Foundation

class PURLoggerConfiguration {
    var logStore: PURLogStore?
    var filterSettings = [PURFilterSetting]()
    var outputSettings = [PUROutputSetting]()
    
    class func defaultConfiguration() -> PURLoggerConfiguration {
        let configuration = PURLoggerConfiguration()
        
        configuration.logStore = PURLogStore(databasePath: nil)
        configuration.filterSettings = []
        configuration.outputSettings = []
        
        return configuration
    }
}
